config/rbac/tekton_role.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tekton-role
```

Comment:
is it okay to change the name to something like tekton-cluster-admin-role like other fies